packages/@react-types/shared/src/dna.d.ts

```diff
@@ -10,6 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
+// This file is generated by lib/varsToTypeScript.js! DO NOT EDIT.
```

Comment:
?

Reply:
generated by running `varsToTypeScript`, not entirely sure if we wanna keep or not.

Reply:
but why wasn't it there previously?

Reply:
dunno, removed in this pull: https://github.com/adobe/react-spectrum/pull/206. Not sure the reason for its removal, seems innocuous to me (helps others understand where the file came from)

Reply:
yeah, i'm definitely in favor of leaving it in if it is truly a generated file